src/atwho.js

```diff
@@ -164,10 +164,14 @@ define([
                 };
             };
 
-            $input.atwho(_atWhoOptions('/data/'));
-            if (options.useRichText) {
-                $input.atwho(_atWhoOptions('#'));
-            }
+            $input.on('focus', function () {
+                if (!$input.data('atwho')) {
```

Comment:
`one` instead of `on` would do the same thing in less code, but that's just a matter of style